closed/custom/common/Modules.gmk

```diff
@@ -41,15 +41,20 @@ else
 	#
 endif
 
+ifeq (false,$(OPENJ9_ENABLE_JFR))
+	MODULES_FILTER += \
+		jdk.jfr \
+		jdk.management.jfr \
+		#
```

Comment:
Outside a rule, a non-continuation line cannot begin with a tab. Please follow the pattern of lines 45-48:
```
  MODULES_FILTER += \
	jdk.jfr \
	jdk.management.jfr \
	#
```